src/js/core.js

```diff
@@ -279,9 +235,21 @@ function MediumEditor(elements, options) {
     }
 
     function shouldAddDefaultAutoLinker() {
```

Comment:
This is unrelated to the goal of this PR, but this shouldn't be `shouldAddDefaultAutoLink` instead? Since you rename AutoLinker to AutoLink?